data-tck/src/main/groovy/io/micronaut/data/tck/tests/AbstractRepositorySpec.groovy

```diff
@@ -3031,6 +3031,54 @@ abstract class AbstractRepositorySpec extends Specification {
             ilikeNames.toSet() == ["Fr_dB1", "Fr_dB2"].toSet()
     }
 
+    void "test data with datetime fields and custom time zone"() {
```

Comment:
Added this test for timezone not in default server timezone that discovered an issue earlier with io.asyncer so think would be good to have it.